spec/mailtrap/suppression_spec.rb

```
@@ -0,0 +1,126 @@
+RSpec.describe Mailtrap::SuppressionsAPI do
```

RuboCop (1.75.5)

[convention] Spec path should end with `mailtrap/suppressions_api*_spec.rb`.

(RSpec/SpecFilePathFormat)